raiden/raiden_service.py

```diff
@@ -730,6 +714,9 @@ def process_claims(self, claims: List[Claim]) -> None:
             fee_config=self.config.mediation_fees,
         )
 
+        # TODO: is this safe or do we need a special state change?
+        chain_state.unresolved_claims.extend(unprocessable_claims)
```

Comment:
Mh, did you base this on an old version? I think this code was removed in the meantime. Maybe @fredo knows more.

Reply:
I did a rebase onto the latest `raiddit` state. Maybe I accidentally reintroduced this.